android/app/build.gradle

```diff
@@ -74,4 +74,8 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation 'io.sentry:sentry-android:1.7.27'
```

Comment:
So is this required to catch the Android platform level stuff? This seems to be an Android Sentry implementation?

Reply:
Yes we want to be able to catch errors that happen in native Android code, either in Flutter framework or plugins, which won't necessarily be reported to Dart.